lib/fog/kubevirt/compute/compute.rb

```diff
@@ -15,6 +15,8 @@ class Compute < Fog::Service
       collection :nodes
       model      :vm
       collection :vms
+      model      :service
```

Comment:
not sure about those naming - service and services are used in fog-core base classes, derived by the providers - we should select a different name for it not to confuse or override other functionality.

Reply:
Correct. There is no naming clash I checked by using our example. The naming is consistent with everything else since we are using kind to name and for this entity we have `kind/Service`. I am happy to change it but please suggest the name.

Reply:
I'd name it k8_service